Dockerfile-MonoRepo

```
@@ -1,29 +1,3 @@
-FROM node:16-alpine
+FROM playsportsgroup/buildtools
 
```

Could you just set this again for the Monorepo image?

```suggestion
ARG CLOUD_SDK_VERSION=410.0.0
ENV CLOUD_SDK_VERSION=$CLOUD_SDK_VERSION
```

---

You set args on the build of the image (IE on the executor) - I didn't think you can set ARGs here on deriving from the image.